examples/cardano-lock-namiwallet/package.json

```diff
@@ -14,10 +14,6 @@
   "dependencies": {
     "@ckb-lumos/lumos": "0.20.0-alpha.0",
     "@emurgo/cardano-message-signing-asmjs": "^1.0.1",
-    "@emurgo/cardano-serialization-lib-asmjs": "^10.0.4",
-    "@types/react": "^17.0.34",
-    "@types/react-dom": "^17.0.11",
-    "react": "^17.0.2",
```

Comment:
cannot remove react, which will make examples not work in Codesandbox, recommend to use yarn resolutions + fixed dependency version